pkg/volumeclaimbinder/types.go

```diff
@@ -139,3 +145,65 @@ func matchStorageCapacity(pvA, pvB *api.PersistentVolume) bool {
 	bSize := bQty.Value()
 	return aSize <= bSize
 }
+
+// allPossibleMatchingAccessModes returns an array of AccessMode arrays that can satisfy a user's requested modes.
+// There are, essentially, 3 ways to request access modes:
+//
+//  RWO -- the most basic form of access (all storage volumes)
+//  RWO+ROX -- 1 writer or many read-only readers (GCEPersistentDisk)
+//  RWO+ROX+RWX -- many writers (NFS)
```

Comment:
I was operating under the assumption that a use requested RWX and that implied RWO and ROX.  Is that not the case?  Does a user have to actually spell out modes they don't want?